wiki/Gameplay/Game_modifier_(lazer)/en.md

```diff
@@ -0,0 +1,110 @@
+- Half Time (HT) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Daycore (DC) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- No Release (NR) ![][osu!mania]
+
+### Difficulty Increase
+
+- Hard Rock (HR) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Sudden Death (SD) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Perfect (PF) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Double Time (DT) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Nightcore (NC) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Fade In (FI) ![][osu!mania]
+- Hidden (HD) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Cover (CO) ![][osu!mania]
+- Flashlight (FL) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Blinds (BL) ![][osu!]
+- Strict Tracking (ST) ![][osu!]
+- Accuracy Challenge (AC) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+### Automation
+
+- Autoplay (AT) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Cinema (CN) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Relax (RX) ![][osu!] ![][osu!taiko] ![][osu!catch]
+- Autopilot (AP) ![][osu!]
+- Spun Out (SO) ![][osu!]
+
+### Conversion
+
+- Target Practice (TP) ![][osu!]
+- Difficulty Adjust (DA) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- [Classic (CL)](/wiki/Gameplay/Game_modifier/Classic) ![][osu!] ![][osu!taiko]
+- Random (RD) ![][osu!] ![][osu!taiko] ![][osu!mania]
+- Mirror (MR) ![][osu!] ![][osu!catch] ![][osu!mania]
+- Alternate (AL) ![][osu!]
+- Swap (SW) ![][osu!taiko]
+- Single Tap (SG) ![][osu!] ![][osu!taiko]
+- Invert (IN) ![][osu!mania]
+- Constant Speed (CS) ![][osu!taiko] ![][osu!mania]
+- Hold Off (HO) ![][osu!mania]
+- 1K, 2K, 3K, 4K, 5K, 6K, 7K, 8K, 9K, 10K ![][osu!mania]
+
+### Fun
+
+- [Transform (TR)](/wiki/Gameplay/Game_modifier/Transform) ![][osu!]
+- [Wiggle (WG)](/wiki/Gameplay/Game_modifier/Wiggle) ![][osu!]
+- [Spin In (SI)](/wiki/Gameplay/Game_modifier/Spin_In) ![][osu!]
+- [Grow (GR)](/wiki/Gameplay/Game_modifier/Grow) ![][osu!]
+- [Deflate (DF)](/wiki/Gameplay/Game_modifier/Deflate) ![][osu!]
+- Wind Up (WU) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Wind Down (WD) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Traceable (TC) ![][osu!]
+- Barrel Roll (BR) ![][osu!]
+- Approach Different (AD) ![][osu!]
+- [Floating Fruits (FF)](/wiki/Gameplay/Game_modifier/Floating_Fruits) ![][osu!catch]
+- [Muted (MU)](/wiki/Gameplay/Game_modifier/Muted) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- [No Scope (NS)](/wiki/Gameplay/Game_modifier/No_Scope) ![][osu!] ![][osu!catch]
+- Magnetised (MG) ![][osu!]
+- Repel (RP) ![][osu!]
+- Adaptive Speed (AS) ![][osu!] ![][osu!taiko] ![][osu!mania]
+- Freeze Frame (FR) ![][osu!]
+- Bubbles (BU) ![][osu!]
+- Synesthesia (SY) ![][osu!]
+- Depth (DP) ![][osu!]
+
+### Other
+
+- No Mod (NM) ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

Comment:
I think this should be done similarly to what the stable article does (giving this term its own section under `Other` and describing the term in a small paragraph.)

The same may go for Touch Device too.

Reply:
touch device is a proper mod and should get an article like any other